microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -3,89 +3,154 @@ description: Version Models used for Microsoft Edge WebView2
 title: Understand WebView2 SDK versions
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 05/06/2021
+ms.date: 08/03/2021
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
 keywords: IWebView2, IWebView2WebView, webview2, webview, wpf apps, wpf, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
 ---
-# Understand WebView2 SDK versions  
+# Understand WebView2 SDK versions
 
-New versions of the WebView2 SDK are shipped at the same general cadence as the Microsoft Edge \(Chromium\) browser, which is approximately every six weeks.  
+<!-- todo: cover Fixed Version.  when/if to specify "the Evergreen Runtime", instead of just "the Runtime".  does it work, contrasting "preview channel" vs. "Runtime" as terminology for mutually exclusive options? -->
 
-## Release and prerelease package  
+The NuGet package for the WebView2 SDK contains both a release and prerelease package.  Either use a prerelease SDK with a preview channel of Microsoft Edge, or use a release SDK with the WebView2 Runtime.
 
-The WebView2 NuGet package contains both a release and pre-release package.  
+*   During development and testing of your WebView2 app, to use experimental WebView2 APIs, use the following combination:
+    *   A _prerelease_ version of the WebView2 SDK.
+    *   A _preview channel_ of Microsoft Edge.
 
-The **release package** is forward compatible and contains the following components.  
+*   For a production release of your WebView2 app, use the following combination:
+    *   A _release_ version of the WebView2 SDK.
+    *   The WebView2 _Runtime_.
 
-*   [Win32 C/C++ APIs][ReferenceWin32]
-*   .NET APIs:  [WPF][DotnetMicrosoftWebWebview2WpfNamespace], [WinForms][DotnetMicrosoftWebWebview2WinformsNamespace], and [Core][DotnetMicrosoftWebWebview2CoreNamespace].  
-    
-APIs in the SDK are fully supported.  
+These two combinations are described below.
 
-The **prerelease package** is a superset of the release package with [Experimental APIs](#experimental-apis).  Avoid using the prerelease SDK to build production apps.  
 
-### Roadmap  
+## Use a prerelease version of the SDK along with a preview channel of Microsoft Edge
```

Comment:
This part might be slightly misleading. It sounds like when you're developing, you use pre-release + non-stable; when you're in prod, you use release + runtime. This isn't the case.

We have pre-release and release packages. Pre-release packages are for developers during development who want to test our latest WebView2 APIs, including the experimental APIs, before its support land in the Runtime. Developers using pre-release packages are recommended to use it against Canary so that the APIs are actually implemented here. 

Release packages only contain stable API. Developers can use them either in development (because not all developers care about the latest experimental APIs that we expose), or in production.